src/chrome/content/rules/francetvinfo.fr.xml

```diff
@@ -35,16 +30,39 @@ Fetch error: http://www.francetvinfo.fr/ => https://www.francetvinfo.fr/: Too ma
 		operation.francetvinfo.fr
 
 -->
-<ruleset name="Franceinfo (partial)" platform="mixedcontent" default_off='failed ruleset test'>
+<ruleset name="Franceinfo (partial)">
 
 	<target host="francetvinfo.fr" />
 	<target host="www.francetvinfo.fr" />
+
+		<!-- Redirect to http -->
+		<exclusion pattern="http://(www\.)?francetvinfo.fr/([\w-/]*)(\.html)?$" />
```

Comment:
Somehow the `rules-test` does not allow `*`, you must use `+` instead. You've forgotten to escape `.` and this `regex` match everything.